src/components/Product/ProductListSection.tsx

```diff
@@ -83,11 +83,7 @@ export function ProductListSection() {
 
       <ProductListWrapper>
         {displayedProducts.map((product, index) => (
-          <ProductItem
-            key={product.id + '-' + index}
-            product={product}
-            rank={index + 1}
-          />
+          <ProductItem key={product.id} product={product} rank={index + 1} />
```

Comment:
사실 제가 이야기 했던 부분은 이런거였어요
```suggestion
          <ProductItem key={product.id} {...product} rank={index + 1} />
```
props를 객체로 넘기는 경우, 객체의 내용은 똑같아도 서로 다른 메모리에 존재할 수 있다보니 메모이제이션이 실패할 수 있답니다.
그런데 이렇게 객체를 해체해서 값으로 넘기는 경우에는 객체가 아니라 값이기 때문에 메모이제이션이 잘 될 확률이 높아요!

Reply:
다만 product가 price랑 brand를 가지고 있다보니, 이런 것들도 잘 정제해서 ProductItem에 넘겨줘야겠죠!?